### ISSUE TITLE
-i to print encoding options used

### ISSUE THREAD

**Issue by matthiaskrgr** [FLIF-hub/FLIF#260]
For flifcrushing it would be useful if "flif -i <file>" could print the encoding options used to encode the flif (maniac tree properties, chance/cutoff, color bucket usage, channel compact, etc...)

**Comment by jonsneyers** [comment]
I agree. Not urgent, and it would require -i to decode more than just the simple header (it would also need to decode the transformation data and the maniac trees), so I propose that it would only do this in combination with -v.

Some of the encode options cannot be derived from the file itself though. In particular, the `--maniac-*` encode parameters cannot be reconstructed, except of course by trial and error :)
